ui/appui-react/src/appui-react/dialog/ModelessDialogManager.tsx

```diff
@@ -96,7 +79,7 @@ export class ModelessDialogManager {
         ModelessDialogManager._idArray.splice(index, 1);
 
       if (ModelessDialogManager.activeDialog === undefined)
-        ModelessDialogManager._topZIndex = ModelessDialogManager.getDialogZIndexDefault();
+        DialogManagerBase.topZIndex = DialogManagerBase.getDialogZIndexDefault();
```

Comment:
If there is no more ModelessDialog active, we reset the `topZIndex`, however, there could still be normal dialog open, which will still want to use the previous `topZIndex`. (same thing in the other way)

Reply:
Right. All the dialogs will be the default zIndex, unless multiple dialogs are open and they get reordered. The reordering is currently broken, so I'll look at that problem when I fix the reodering.

Probably, we want to move the whole thing down to the DialogManagerBase class.